web/src/app/components/dashboard/dashboard.component.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../../styles/variables.scss";
+
+.cards {
+    background: #F9F9F9;
+
+    .mat-card-title {
+        font-size: 0.87rem !important;
```

Comment:
Can we avoid using `!important` as can cause issues in the future

Reply:
okay Eddie